.github/workflows/go.yml

```diff
@@ -74,3 +74,23 @@ jobs:
 
     - name: Run vHive tests that need manual cleaning
       run: make test-man-bench
+
+  test-profile:
+    name: Profile integration tests
```

Comment:
These are not integration tests, these are unit tests.

Reply:
Done